### PR TITLE
Refined Readme Introduction To Be More Exciting

### PR DESCRIPTION
**Pull request by clearsitedesigns**
update readme for user-focused content. Add descriptive table card explaining floating points on common graphics cards."

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,21 @@
 # Axolotl
 
-Axolotl is a tool designed to streamline the fine-tuning of various AI models, offering support for multiple configurations and architectures.
+Unleash the true potential of advanced language models and AI using Axolotl, the ultimate solution for effortlessly refining a wide range of AI models. Tailored to suit your specific requirements, Axolotl offers adaptable configurations encompassing popular models such as Llama, Pythia, and more. Supercharge your training journey through mixed precision and memory optimization, guaranteeing quicker outcomes without sacrificing quality. Seamlessly manage diverse data formats, simplifying the data preparation process.
```
Comment (winglian) [inline README.md:3]: Maybe say customizable instead of tailored? Tailored implies we handle the customization part?

**Review by winglian — APPROVED** [review]
Looks good to me. @tmm1 @NanoCode012

File: README.md (hunk 1 above)
Comment (tmm1) [reply README.md:3]: Was this generated?

I prefer the simpler language that was in place before. My ambitions are simply to build a good tool, not some ultimate solution.

I think this kind of content would be better suited to a homepage which can use more marketing speak.

Comment (NanoCode012) [reply README.md:3]: I agree with the above. I think it’s too verbose. Perhaps the bullet point description of the features is better/enough?

Comment (clearsitedesigns) [reply README.md:3]: I was asked to make this more exciting and more understandable. As an English professional, your intro doesn't even give people a clue about what this is for. So I made it actually sound like a product people would want to try and use. Don't assume people understand this world.

Comment (NanoCode012) [reply README.md:3]: Hey @clearsitedesigns , I understand where you're coming from. Would we be able to see if a middle ground would work? I have provided a sample "Features" below. My inspiration is from https://github.com/oobabooga/text-generation-webui which I find easy to understand and follow.

```
Unleash the true potential of advanced language models and AI using Axolotl, the ultimate solution for effortlessly refining a wide range of AI models. 
Dive into the features below to unlock the full potential of your AI endeavors.

- train a wide variety of models from llama to pythia
- support fullfinetune, lora, qlora, and gptq
- customize the configuration using a simple yaml file or overwrite with cli
- load various kinds of dataset format, use your own format, or bring your own tokenized dataset
- integrated with xformer, flash attention, and multipacking
- works with single GPU or multi-gpu with FSDP or Deepspeed
- run easily with Docker locally or on a cloud service
- and more!
```

What do you think? Would it be possible to have a simple short line description followed by bulletpoints features? Of course, the wording can definitely be improved, I just tried to list the major features.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,21 @@
 # Axolotl
 
-Axolotl is a tool designed to streamline the fine-tuning of various AI models, offering support for multiple configurations and architectures.
+Unleash the true potential of advanced language models and AI using Axolotl, the ultimate solution for effortlessly refining a wide range of AI models. Tailored to suit your specific requirements, Axolotl offers adaptable configurations encompassing popular models such as Llama, Pythia, and more. Supercharge your training journey through mixed precision and memory optimization, guaranteeing quicker outcomes without sacrificing quality. Seamlessly manage diverse data formats, simplifying the data preparation process. 
+
+With Axolotl, configuring model predictions is a breeze, allowing you to assess models during and after training. Join a dynamic community of AI enthusiasts, foster collaboration, and benefit from shared experiences. Embrace the efficiency of gradient checkpointing during training and seamlessly integrate with the Hugging Face Transformers ecosystem. Embrace the Axolotl advantage, and take your AI initiatives to unparalleled heights.
```
Comment (mhenrichsen) [inline README.md:5]: Not exactly sure what "configuring model predictions" means.

File: README.md [hunk 3]
```diff
@@ -1,6 +1,21 @@
...
+  <li>Versatile configurations for Llama2, Llama, Pythia, and more.</li>
+  <li>Speed up training via precision and memory optimization.</li>
+  <li>Seamlessly handle diverse data formats with ease.</li>
+  <li>Simplify model prediction setup and evaluation.</li>
+  <li>Collaborate and learn from a vibrant AI community.</li>
+  <li>Integrate smoothly with Hugging Face Transformers.</li>
+  <li>Unlock AI's potential with Axolotl's prowess.</li>
+  <li>Boost training efficiency and outcome quality.</li>
+  <li>Test model inference via console effortlessly.</li>
+</ul>
```
Comment (mhenrichsen) [inline README.md:18]: I feel like a lot of these say the same thing, but in different wording.

Comment (clearsitedesigns) [reply README.md:18]: That how you get people to understand things. Repetition,

**Pull request closed without merging**